rm/src/main/java/io/seata/rm/AbstractRMHandler.java

```diff
@@ -105,6 +105,13 @@ protected void doBranchCommit(BranchCommitRequest request, BranchCommitResponse
             LOGGER.info("Branch commit result: " + status);
         }
 
+        for (GlobalTransactionHook hook : GlobalTransactionHookManager.popHooks(xid)) {
```

Comment:
How about remove the GlobalTransactionHookManager, and make it to orther pr?

Reply:
ok